internal/cmd/controller/options/leaderelection.go

```diff
@@ -0,0 +1,50 @@
+package options
```

Comment:
That options package is for bundledeployment options, I think.
> Package options merges the BundleDeploymentOptions, so that targetCustomizations take effect.